src/bun.js/node/node_fs_watcher.zig

```diff
@@ -403,6 +404,32 @@ pub const FSWatcher = struct {
 
                             current_task.append(relative_path, event_type, .destroy);
 
+                            const full_path = bun.default_allocator.dupeZ(u8, path_slice) catch unreachable;
+                            const is_watched = blk: {
+                                for (this.file_paths.slice()) |path| {
```

Comment:
This will get very expensive fast. It sounds like we should have a hash table here.

Reply:
Should I change the data structure to ArrayHashMap?

https://ziglang.org/documentation/master/std/#A;std:ArrayHashMap

Reply:
Should I change the data structure to ArrayHashMap?

https://ziglang.org/documentation/master/std/#A;std:ArrayHashMap

Reply:
We should probably use `std.hash.Wyhash` and `std.HashMap` take a look on https://github.com/oven-sh/bun/blob/main/src/bun.zig#L826-L828

Reply:
Done.